front-end/src/Components/Employee/EmployeesList/employee.resource.tsx

```diff
@@ -0,0 +1,31 @@
+import { config } from '../../Config/config';
+
+export const getAllEmployees = async () => {
+  return await fetch(config + `/AllEmployees`)
+    .then((response) => {
+      return response.json();
+    })
+    .then((res: any) => {
+      return res;
+    })
+    .catch((error: any) => {
+      console.log(error);
```

Comment:
we could handle errors better here, should return a value indicating an error has occurred